README.md

```diff
@@ -62,7 +62,7 @@ Resources:
 ###Class: networkmanager
 
 ####`enable`
-Should the service be enabled during boot time ? Defaults to `true`.
+Should the service be enabled during boot time ? Default to `true`.
```

Comment:
These are not errors!
http://conjugator.reverso.net/conjugation-english-verb-default.html